### PR TITLE
Add since annotation to all PHPdoc blocks

### LINKED ISSUE #3873: Add since annotation to all doc blocks
## Feature Description

With stable around the corner, I think it is time to add `@since` notation to all PHP docblocks. 

## Additional Context

There are currently 203 functions / methods in includes
There are currently 227 functions / methods in tests/phpunits

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

### PR DESCRIPTION
**Pull request by spacedmonkey**
## Summary

Add `@since` annotation.

## Testing Instructions

---

Fixes #3873

### CODE REVIEW

File: includes/Media.php [hunk 1]
```diff
@@ -312,7 +319,9 @@ public function rest_api_init() {
 	/**
 	 * Force media attachment as string instead of the default array.
 	 *
-	 * @param array $prepared Prepared data before response.
+	 * @since 1.0.0
+	 *
+	 * @param array $prepared Prepared data before response.s
```
Comment (swissspidy) [inline includes/Media.php]:
```suggestion
	 * @param array $prepared Prepared data before response.
```

Comment (spacedmonkey) [reply includes/Media.php]: Fixed

File: includes/Activation_Flag.php [hunk 2]
```diff
@@ -73,9 +81,12 @@ public function get_activation_flag( $network_wide = false ) {
 	/**
 	 * Deletes the flag that the plugin has just been activated.
 	 *
-	 * @since 1.10.0 Migrated from Activation class.
+	 * Migrated from Activation class.
```
Comment (spacedmonkey) [inline includes/Activation_Flag.php]: @swissspidy Not sure what this comment was.

Comment (swissspidy) [reply includes/Activation_Flag.php]: No idea either 🤷  Probably added by accident

File: includes/Activation_Flag.php [hunk 3]
```diff
@@ -73,9 +81,12 @@ public function get_activation_flag( $network_wide = false ) {
...
+	 *
+	 * @since 1.0.0
```
Comment (spacedmonkey) [inline includes/Activation_Flag.php]:
```suggestion
	 * @since 1.0.0
```

**Review by swissspidy — APPROVED** [review]

**Comment by spacedmonkey** [comment]
@swissspidy Should we merge this?

**Pull request merged**